src/insights_client/__init__.py

```diff
@@ -237,9 +237,14 @@ def _main():
             sys.exit('Unable to load Insights Config')
 
         if config["version"]:
-            from insights_client.constants import InsightsConstants as constants
-            print("Client: %s" % constants.version)
-            print("Core: %s" % InsightsClient().version())
+            try:
+                from insights_client.constants import InsightsConstants
+            except ImportError:
+                # The source file is build from 'constants.py.in' and is not available during development
+                class InsightsConstants(object):
+                    version = "upstream"
```

Comment:
nitpick: more than "upstream" I'd say "development": you will get the same also when trying to run an old version that is not necessarily "upstream"